.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+## Before submitting
+
+Before submitting a PR, please make sure to check our contributing guidelines regarding [code formatting](https://github.com/pytorch/vision/blob/main/CONTRIBUTING.md#code-formatting-and-typing), [tests](https://github.com/pytorch/vision/blob/main/CONTRIBUTING.md#unit-tests), and [documentation](https://github.com/pytorch/vision/blob/main/CONTRIBUTING.md#documentation).
```

Comment:
If we keep this as-is, it will show up as rendered text on every single PR and this will create noise. I think we should wrap all of these in html comments so that we don't see these comments on the opened PRs. This might mean that we need to inline the links

Reply:
done :]